src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -683,12 +681,12 @@ public struct ReplicationItem : IDisposable
             #region Document
 
             public string Id;
-            public int Position;
             public string ChangeVector;
             public int DocumentSize;
             public string Collection;
             public long LastModifiedTicks;
             public DocumentFlags Flags;
+            public BlittableJsonReaderObject Document;
```

Comment:
This struct is disposable. We probably need to dispose this?

Reply:
Added disposal for it.
Pretty sure that this is disposed during context dispose anyway.

Reply:
We do dispose it in MergedDocumentReplicationCommand